src/install_update.ts

```diff
@@ -31,22 +32,28 @@ export interface Update {
  * updatable releases are found, dependency files are updated with the
  * latest dependency versions and the project changes are cached.
  */
-export const updateDependencies = async () => {
+export const updateDependencies = async (hookCLI: Protocol) => {
   const { releases } = await checkForSDKUpdates();
   const updatableReleases = releases.filter((r) =>
     r.update && r.current && r.latest
   );
-  const updateResp = await createUpdateResp(updatableReleases);
+  const updateResp = await createUpdateResp(updatableReleases, hookCLI);
 
   // If no errors occurred during installation, re-build
   // project as a means to cache the changes
   if (!updateResp.error) {
+    hookCLI.log(
+      "install-update successfully updated dependencies, running build hook to cache dependencies...",
+    );
     try {
       // TODO :: This try/catch should be nested within createUpdateResp
       // but doing so surfaces an issue with the --allow-run flag not
       // being used, despite its presence and success at this level
       runBuildHook();
     } catch (err) {
+      hookCLI.error(
```

Comment:
Same question about additional log line

Reply:
No reason, exercising the diagnostic information. Lemme know if I should remove it.